depends/patches/zeromq/0001-fix-build-with-older-mingw64.patch

```diff
@@ -1,17 +1,17 @@
-From 1a159c128c69a42d90819375c06a39994f3fbfc1 Mon Sep 17 00:00:00 2001
```

Comment:
Given that these are just being updated for upstream, you could leave the original From/Date

Reply:
the date didn't change. the thing that changed was the hex value, which is only the local commit made to my libzmq repo so that i could run `git format-patch master --stdout` to get a patch that had most of the same formatting as the original one. it's a pretty meaningless line. i was actually thinking of just using the `git diff` output for the patch file contents as that would make these diffs easier in the future, but then you lose the commit comment being in the patch file.